views/github-create-branch.hbs

```diff
@@ -30,26 +30,28 @@
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
+
   <div class="gitHubCreateBranch__title">
     <div class="gitHubCreateBranch__header">Create GitHub Branch</div>
-    <div class="gitHubCreateBranch__subHeader">
-      Creating a branch for <b>{{issue.key}}</b>
-    </div>
+    <div class="gitHubCreateBranch__subHeader skeleton skeleton-text">Creating a branch for <b>{{issue.key}}</b></div>
```

Comment:
For most elements, simply adding skeleton and skeleton-element will make it possible implement this loading state.